install-steps.md

```diff
@@ -577,6 +603,9 @@ oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --t
         baremetal: {}
     platform:
       baremetal:
+        # If you are using cached images you need to configure the following properties (bootstrapOSImage and clusterOSImage) so the installer gets the images from the cache https://github.com/openshift-kni/baremetal-deploy/blob/master/install-steps.md#create-rhcos-images-cache-optional
+        # bootstrapOSImage: http://172.22.0.1/$RHCOS_QEMU_URI?sha256=$RHCOS_QEMU_SHA_UNCOMPRESSED
```

Comment:
If we go the podman route the link changes to http://172.22.0.1:8080 as we map port 8080